common/lcmInputs.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import "time"
+
+type InputMsgType uint16
+
+var EInputMsgType InputMsgType
+
+func(InputMsgType) Invalid()                                InputMsgType {return InputMsgType(0)}
+func(InputMsgType) CancelJob()                              InputMsgType {return InputMsgType(1)}
+func(InputMsgType) E2EInterrupts()                          InputMsgType {return InputMsgType(2)}
+func(InputMsgType) ThroughputAdjustment()                   InputMsgType {return InputMsgType(3)}
```

Comment:
Consider making ThroughputAdjustment more future proof, ex: PerformanceAdjustment

Reply:
PerformanceAdjustment's body could be a JSON that includes many fields

Reply:
Introduced a struct to handle this

Reply:
Introduced a struct to handle this